docs/developer/api.rst

```diff
@@ -34,6 +34,17 @@ Indexing Strategies
 
 GeoMesa uses several different strategies to index simple features. In the code, these strategies are abstracted as 'tables'. For details on how GeoMesa encodes and indexes data, see tables. For details on how GeoMesa chooses and executes queries, see the ``org.locationtech.geomesa.accumulo.index.QueryPlanner`` and ``org.locationtech.geomesa.accumulo.index.QueryStrategyDecider`` classes.
 
+Explaining:  Query Plans
+++++++++++++++++++++++++
+
+Given a data store and a query, you can ask GeoMesa to explain its plan for how to execute the query:
+
+.. code-block:: java
+
+    dataStore.explainQuery(query)
+
+Knowing the plan -- including information such as the indexing strategy -- can be useful when you need to debug slow queries.  It can suggest when indexes should be added as well as when query-hints may expedite execution times.
+
```

Comment:
It would be nice to also link here to the part in the docs for enabling explain logging in geoserver.